Build problem on MacOS with OpenJDK 13

I received a bug report that, when using MacOS and OpenJDK 13, compilation fails with the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:testCompile (default-testCompile) on project SQLancer: Compilation failure: Compilation failure: 
[ERROR] GitHub/sqlancer/test/sqlancer/dbms/TestConfig.java:[3,14] Implicit super constructor java.lang.Object() is undefined for default constructor. Must define an explicit constructor
[ERROR] GitHub/sqlancer/test/sqlancer/dbms/TestConfig.java:[4,25] String cannot be resolved to a type
[ERROR] GitHub/sqlancer/test/sqlancer/dbms/TestConfig.java:[5,25] String cannot be resolved to a type
[ERROR] -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -52,6 +52,11 @@
             <artifactId>plexus-compiler-eclipse</artifactId>
             <version>2.8.6</version>
           </dependency>
+          <dependency>
+            <groupId>org.eclipse.jdt</groupId>
+            <artifactId>ecj</artifactId>
+            <version>3.22.0</version>
+          </dependency>
         </dependencies>
       </plugin>
       <plugin>

